README.md

```diff
@@ -32,7 +32,11 @@ $ find . -name '*.go' -not -path './vendor*' -exec go fmt {} \;
 
 ## Import data to database
 
-You need to have a database name "breizhtrip_development".
+You need to have a database name:
+* "breizhtrip_development" or
+* "reizhtrip_production" or
+* "breizhtrip_test"
+
```

Comment:
Suggest edit:
You need a database named "breizhtrip_" + `environment name`. It can be `development`, `test` or `production`